### PR TITLE
chore: Replace xo with eslint

### PR DESCRIPTION
**Pull request by Tobi-mmt**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
 	],
+	"plugins": [
+		"react",
+		"@typescript-eslint",
+		"prettier"
+	],
+	"env": {
+		"browser": true,
+		"jasmine": true,
+		"jest": true
```
Comment (mismosmi) [inline .eslintrc:16]: jasmine _and_ jest?

File: .eslintrc [hunk 2]
```diff
@@ -1,11 +1,30 @@
...
+		"react": {
+			"pragma": "React",
+			"version": "detect"
+		}
+	},
+	"parser": "@typescript-eslint/parser",
 	"rules": {
-		"react-hooks-nesting": "error",
-		"jsx-boolean-value": false
+		"@typescript-eslint/ban-ts-comment": "warn",
```
Comment (mismosmi) [inline .eslintrc:26]: I assume we can get rid of this if we fix whatever needs the `@ts-ignore` comments?

File: package.json [hunk 3]
```diff
@@ -54,17 +48,16 @@
     "@typescript-eslint/parser": "^4.21.0",
     "babel-loader": "^8.2.2",
     "editorconfig-checker": "^4.0.2",
-    "eslint-config-xo": "^0.38.0",
-    "eslint-config-xo-react": "^0.25.0",
-    "eslint-config-xo-typescript": "^0.38.0",
-    "eslint-plugin-react": "^7.23.1",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
```
Comment (mismosmi) [inline package.json:52]: As far as I see this is not actually extended in eslintrc. Does `plugin:prettier/recommended` automatically add this? And if so, do we need to still keep it as an explicit dependency?

File: src/components/arrow/arrow.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import React from 'react';
```
Comment (mismosmi) [inline src/components/arrow/arrow.tsx:1]: I will just assume that all changes in `src` are eslint/prettier auto-fixes and not read those as thoroughly 🙈

File: .eslintrc (hunk 2 above)
Comment (Tobi-mmt) [reply .eslintrc:26]: true, but don't know how to get rid of the ts issues 😅

File: .eslintrc (hunk 1 above)
Comment (Tobi-mmt) [reply .eslintrc:16]: transfer mistake^^ think also we can remove jasmine

File: .eslintrc (hunk 2 above)
Comment (mismosmi) [reply .eslintrc:25]: 🤷

**Pull request merged**